action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'BrowserStack Actions'
```

Comment:
This is duplicate file of setup-env/action.yml

Reply:
We would be publishing to the Marketplace with this yml file. Its the parent action file which in turn calls the `setup-env` action internally. More like a symlink so that the action that we end up publishing also triggers something than just being a repository.
And we can't publish an action if it doesn't have the `action.yml` file.

Reply:
Resolving this.